framework/test/fixtures/config/devnet/config.json

```diff
@@ -439,7 +439,7 @@
 			}
 		},
 		"network": {
-			"list": [
+			"seedPeers": [
 				{
 					"ip": "127.0.0.1",
```

Comment:
Do we want to keep the `ip` field or rename it to `ipAddress` to match the p2p library?
Either way is OK for me, just something to think about.